### DISCUSSION TITLE
Version control the generated types?

### DISCUSSION THREAD

**Discussion by mittalyashu** [go-jet/jet#399] [Q&A]
Do we have to version control the generated types via Jet CLI or we can add that directory to `.gitignore`.

since, for local development we have different multiple db's with different names and in the types, the schema name is explicitly added.

```go
var Posts = newPostsTable("feature-1-branch", "posts", "")
```

**Comment by houtn11** [comment]
For [multi-tenant environment](https://github.com/go-jet/jet/wiki/FAQ#how-to-use-jet-in-multi-tenant-environment) you can change the schema name by calling on server startup: 

```golang
table.UseSchema("schema_name")
view.UseSchema("schema_name")
```

**Reply by mittalyashu** [reply]
What about version controlling the types generated by Jet CLI?

**Reply by safaci2000** [reply]
Personally I would store the datamodels as close to where your database definition live.  For my current project the same application manages the DDLs and applies the SQL changes as part of the release.  So I keep the datamodels in the same repo.

If there was a different repo that would manage that, I would have CICD re-generate the models and basically create a 'release' of a library that I import into my project that reflects the database changes.
